src/Config/Environment.php

```diff
@@ -113,6 +118,16 @@ public function getEnvVarName(string $name): string
         return $this->systemConfig->get($name);
     }
 
+    /**
+     * Get a value of the environment variable MAGE_ERROR_REPORT_DIR_NESTING_LEVEL.
+     *
+     * @return int|null|string
```

Comment:
Method getEnv has more return types. `@return array|string|int|null|bool`

Maybe is better to convert the result of this function to integer before return?

Reply:
@oshmyheliuk 
No need. We present only information about the value of this environment variable. And we show this value as it is.

Reply:
Ok, but in this case, return types should be the same as in the `getEnv` method.

Reply:
Done